packages/rs-drive-abci/src/platform_types/snapshot/mod.rs

```diff
@@ -168,4 +207,32 @@ mod tests {
             fs::remove_dir_all(grove_dir.path()).unwrap();
         }
     }
+
+    #[test]
+    fn test_offer_snapshot() {
+        let grove_dir = tempfile::tempdir().unwrap();
+        let replication_dir = tempfile::tempdir().unwrap();
+        let grove = GroveDb::open(grove_dir.path()).unwrap();
+        let mut manager = Manager::new("".to_string(), Some(3), Some(1));
```

Comment:
My concern is that this will lead to writing files in a current working dir, which is not what we want. If it doesn't matter now, it can matter in future. 

So, as a good practice, we should initialize correctly, with a tempdir.